packages/perspective-phosphor/src/ts/widget.ts

```diff
@@ -368,7 +368,7 @@ export class PerspectiveWidget extends Widget {
         if (!viewer.notifyResize) {
             console.warn("Warning: not bound to real element");
         } else {
-            const resize_observer = new MutationObserver(viewer.notifyResize.bind(viewer));
+            const resize_observer = new MutationObserver(() => viewer.notifyResize.call(viewer));
```

Comment:
We probably don't need this block of code. Phosphors `onUpdateRequest` lifecycle method can probably replace this

Reply:
Noted - I've removed this from the minimal-theme branch